plasmapy/transport/classical/epperlein_haines.py

```diff
@@ -0,0 +1,238 @@
+α1pp = np.array([2.53] * 15)
+a0pp = np.array([3280, 3720, 3790, 3670, 3370, 3280, 3250, 3200, 3270, 3390, 3360, 3360, 3520, 3720, 3540])
+a1pp = np.array([3460, 6690, 8990, 10200, 10600, 11100, 11700, 12200, 13300, 14700, 15100, 16200, 18500, 20800, 21900])
+a2pp = np.array([366, 554, 675, 735, 744, 769, 793, 825, 863, 925, 940, 985, 1080, 1180, 1180])
+
+# TABLE III
+#β0(Z=20) is clearly incorrect; replaced with fitted value
+β0 = np.array([0.702, 0.9054, 1.018, 1.092, 1.146, 1.186, 1.218, 1.244, 1.283, 1.312, 1.334, 1.3801, 1.414, 1.455, 1.5])
+β0p = np.array([1.05e3, 1.38e3, 1.55e3, 1.64e3, 1.71e3, 1.74e3, 1.73e3, 1.79e3, 1.92e3, 1.89e3, 1.92e3, 2.01e3, 2.09e3, 2.16e3, 2.20e3])
+β1p = np.array([6.33] * 15)
+b0p = np.array([3710, 3800, 3800, 3740, 3720, 3640, 3530, 3570, 3740, 3580, 3580, 3620, 3680, 3690, 3650])
+b1p = np.array([4110, 7050, 8750, 9730, 10700, 11100, 11300, 11700, 12900, 13000, 13300, 14300, 15200, 16000, 16800])
+b2p = np.array([515, 642, 690, 707, 731, 735, 729, 734, 772, 762, 765, 789, 808, 822, 809])
+β0x = np.array([0.8831, 1.812, 2.589, 3.236, 3.78, 4.243, 4.64, 4.986, 5.556, 6.007, 6.372, 7.144, 7.874, 8.756, 9.844])
+β0pp = np.array([2.54, 4.40, 3.77, 3.43, 3.20, 3.05, 2.92, 2.82, 2.70, 2.62, 2.55, 2.43, 2.31, 2.26, 2.15])
+β1pp = np.array([1.5] * 15)
+b0pp = np.array([2.87, 2.43, 1.46, 1.06, .848, .718, .629, .565, .486, .436, .401, .341, .294, .258, .219])
+b1pp = np.array([3.27, 5.18, 4.34, 3.92, 3.66, 3.48, 3.33, 3.21, 3.09, 3.00, 2.93, 2.81, 2.68, 2.64, 2.53])
+b2pp = np.array([7.09, 9.34, 8.65, 8.27, 8.02, 7.83, 7.68, 7.55, 7.41, 7.30, 7.22, 7.07, 6.91, 6.84, 6.72])
+
+
+# TABLE IV
+γ0 = np.array([3.203, 4.931, 6.115, 6.995, 7.68, 8.231, 8.685, 9.067, 9.673, 10.13, 10.5, 11.23, 11.9, 12.67, 13.58])
+γ0p = np.array([6.18, 9.30, 10.2, 9.14, 8.60, 8.57, 8.84, 7.93, 7.44, 7.32, 7.08, 6.79, 6.74, 6.36, 6.21])
+γ1p = np.array([4.66, 3.96, 3.72, 3.6, 3.53, 3.49, 3.49, 3.43, 3.39, 3.37, 3.35, 3.32, 3.3, 3.27, 3.25])
+c0p=np.array([1.93, 1.89, 1.66, 1.31, 1.12, 1.04, 1.02, 0.875, 0.77, 0.722, 0.674, 0.605, 0.566, 0.502, 0.457])
+c1p=np.array([2.31, 3.78, 4.76, 4.63, 4.62, 4.83, 5.19, 4.74, 4.63, 4.7, 4.64, 4.65, 4.81, 4.71, 4.81])
+c2p= np.array([5.3, 7.78, 8.88, 8.8, 8.8, 8.96, 9.24, 8.84, 8.71, 8.73, 8.65, 8.6, 8.66, 8.52, 8.53])
+γ0x = np.array([6.071, 15.75, 25.65, 34.95, 43.45, 51.12, 58.05, 64.29, 75.04, 83.93, 91.38, 107.8, 124.3, 145.2, 172.7])
+γ0pp = np.array([4.01, 2.46, 1.13, 0.628, 0.418, 0.319, 0.268, 0.238, 0.225, 0.212, 0.202, 0.200, 0.194, 0.189, 0.186])
+γ1pp = np.array([2.5] * 15)
+c0pp = np.array([.661, .156, .0442, .018, .00963, .00625, .00461, .00371, .003, .00252, .00221, .00185, .00156, .0013, .00108])
+c1pp = np.array([.931, .398, .175, .101, .0702, .0551, .0465, .041, .0354, .0317, .0291, .0256, .0228, .0202, .018])
+c2pp = np.array([2.5, 1.71, 1.05, .775, .646, .578, .539, .515, .497, .482, .471, .461, .45, .44, .43])
```

Comment:
Yup, that's definitely transport theory!

Reply:
One model down...many to go 🤦